src/DatePicker/datePicker.css

```diff
@@ -4,16 +4,21 @@
 
 .default
 {
-    width:  auto;
+    width:          auto;
+    box-shadow:     0 4px 10px 0 color( var( --PC-GREY ) a(10%) );
```

Comment:
`box-shadow` should not be added here.  It should be handled by the Dropdown component instead.